data/theme/gnome-shell-sass/_endless.scss

```diff
@@ -845,7 +845,7 @@ popup-separator-menu-item {
     }
 }
 
-// PayGo unlock screen (based on .login-dialog)
+// Pay-As-You-Go unlock screen (based on .login-dialog)
```

Comment:
Nitpick: I think the Camel-And-Hyphens style is the ugliest way to write it, so I'd prefer "Pay As You Go" (here and the case below) instead